### PR TITLE
Modern times

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**

### CODE REVIEW

File: package.exs [hunk 1]
```diff
@@ -1,19 +0,0 @@
-defmodule Poolgirl.Mixfile do
```
Comment (paulo-ferraz-oliveira) [inline package.exs:1]: I'm not sure this file is needed.

File: bench/basho_bench_driver_poolgirl.config [hunk 2]
```diff
@@ -1,21 +0,0 @@
-{driver, basho_bench_driver_poolgirl}.
```
Comment (paulo-ferraz-oliveira) [inline bench/basho_bench_driver_poolgirl.config:1]: The benchmark code is broken, unrequired and pulls a lot of crap just for a simple test.

File: package.exs (hunk 1 above)
Comment (g-andrade) [reply package.exs:1]: This file is needed in order to publish to Hex.

Alternatively, there's `rebar3_hex` plugin (which is likely a better option.)

**Review by g-andrade — CHANGES_REQUESTED** [review]
The lib won't be publishable to Hex with the changes as they are.

**Comment by paulo-ferraz-oliveira** [comment]
> The lib won't be publishable to Hex with the changes as they are.

Sure it will. Just install `rebar3_hex` globally, in your system, lest we have to always keep updating this lib. for all the projects, which I prefer not to.

**Comment by paulo-ferraz-oliveira** [comment]
I'll add this plugin later, directly in `master`. For the time being (and since we'll want to publish this, which will mean the plugin "should" be there) I propose we merge.

**Pull request merged**